scripts/Logistics-Pipes.zs

```diff
@@ -843,8 +848,45 @@ recipes.addShaped(DisconnectWest, [
 [<ore:craftingToolScrewdriver>, DisconnectUp, null],
 [null, null, null]]);
 
+
 // ||||| Fluid Pipes |||||
-//Disabled for right now!
+Assembler.addRecipe(<LogisticsPipes:item.logisticsFluidContainer>, [<miscutils:gt.Volumetric_Flask_8k>, <ore:circuitBasic>, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.lead> * 144, 1600, 120);
+Assembler.addRecipe(<LogisticsPipes:item.logisticsFluidContainer>, [<miscutils:gt.Volumetric_Flask_8k>, <ore:circuitBasic>, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.tin> * 72, 1600, 120);
+Assembler.addRecipe(<LogisticsPipes:item.logisticsFluidContainer>, [<miscutils:gt.Volumetric_Flask_8k>, <ore:circuitBasic>, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.solderingalloy> * 36, 1600, 120);
+
+
+// --- Basic Fluid Pipe
+Assembler.addRecipe(<LogisticsPipes:item.PipeFluidBasic>, [<LogisticsPipes:item.PipeItemsBasicLogistics>, <LogisticsPipes:item.logisticsFluidContainer>, <gregtech:gt.metaitem.01:32611>, <gregtech:gt.metaitem.01:32651>], <liquid:molten.lead> * 144, 1600, 120);
+Assembler.addRecipe(<LogisticsPipes:item.PipeFluidBasic>, [<LogisticsPipes:item.PipeItemsBasicLogistics>, <LogisticsPipes:item.logisticsFluidContainer>, <gregtech:gt.metaitem.01:32611>, <gregtech:gt.metaitem.01:32651>], <liquid:molten.tin> * 72, 1600, 120);
+Assembler.addRecipe(<LogisticsPipes:item.PipeFluidBasic>, [<LogisticsPipes:item.PipeItemsBasicLogistics>, <LogisticsPipes:item.logisticsFluidContainer>, <gregtech:gt.metaitem.01:32611>, <gregtech:gt.metaitem.01:32651>], <liquid:molten.solderingalloy> * 36, 1600, 120);
+
+// --- Fluid Supplier Pipe
+Assembler.addRecipe(<LogisticsPipes:item.PipeFluidSupplierMk2>, [<LogisticsPipes:item.PipeFluidBasic>, <LogisticsPipes:item.itemModule:502>, <LogisticsPipes:item.logisticsFluidContainer>, <gregtech:gt.integrated_circuit:18> * 0], null, 600, 120);

```

Comment:
Basic fluid pipe already used one logistic fluid container in its recipe, why add another here? It's not like it's a huge deal but I want to hear your idea. I could be missing something.

Reply:
Removed the Fluid container item